setup.py

```diff
@@ -117,11 +117,12 @@ def _path_under_setup(*args):
     url=url,
     license=license,
     packages=[pkg_name] + tests,
-    install_requires=['numpy'] + (['cython'] if USE_CYTHON else []),
-    setup_requires=['numpy'] + (['cython'] if USE_CYTHON else []),
-    extras_require={'docs': ['Sphinx', 'sphinx_rtd_theme', 'numpydoc']},
+    include_dirs=['levmar/levmar-2.6', np.get_include()],
     ext_modules=ext_modules,
-    zip_safe=False,
+    data_files=[('lib/site-packages/' + pkg_name, glob('libs/*.dll'))],
+    # cmdclass = {'build_ext': build_ext},
+    install_requires=['numpy', 'nose', 'icc_rt'],
```

Comment:
'icc_rt' is Intel's C Compiler runtime? That's no universal requirement either....

Reply:
icc_rt includes binaries that are part of Intel Redistributable package, which are required by the pre-built LAPACK binaries, as given in https://www.fi.muni.cz/~xsvobod2/misc/lapack/.